doc/api/async_hooks.md

```diff
@@ -272,24 +273,25 @@ elaborate to make calling context easier to see.
 
 ```js
 const async_hooks = require('async_hooks');
+const fs = require('fs');
 
 let indent = 0;
 async_hooks.createHook({
   init(asyncId, type, triggerId) {
     const cId = async_hooks.currentId();
-    fs.writeSync(1, ' '.repeat(indent) + `${type}(${asyncId}): ` +
+    fs.writeSync(1, `${' '.repeat(indent)}${type}(${asyncId}): ` +
```

Comment:
I agree with @AndreasMadsen , the previous version was more readable in my eyes.